libproject/devstart.rb

```diff
@@ -37,6 +40,115 @@ def swagger_regen()
   :fn => Proc.new { |*args| swagger_regen(*args) }
 })
 
+def capitalize(str)
+   str.split('_').map {|x| x.capitalize}.join
+end
+
+ID_SUFFIX = "_id"
+
+def remove_id(id_column)
+  if not id_column.end_with? ID_SUFFIX then
+    raise "Foreign key did not end with " + ID_SUFFIX
+  end
+  return id_column[0..(id_column.length - ID_SUFFIX.length - 1)] 
+end
+
+def escape(str)
+  if str then
+    return str.inspect
+  end
+  return '""'
+end
+
+def column_dict(column)
+  return '{"Name": "' + column['name'] + 
+           '", "Type": "' + column['type'] + '", "Description": ' + 
+           escape(column['description']) + '}'
+end
+
+def write_tables(f, tables, add_to_cohort_tables)   
+  tables.each do |table_name, table_dict|
+    table_columns = table_dict['columns']
+    f.puts('class ' + capitalize(table_name) + '(object):')    
+    table_columns.each do |column|
+      column_name = column['name']
+      f.puts('  ' + column_name + ' = "' + column_name + '"')           
+    end            
+    f.puts('  table_name = "' + table_name + '"')
+    f.puts('  columns = pd.DataFrame([' + 
```

Comment:
How do you imagine this being used? Is this just for easy tabular schema access, e.g. as documentation?

Reply:
Yeah, you can then say just:

print Person.columns

And you get a nice table of documentation on the columns. Added a comment to the README on this.